openprescribing/matrixstore/build/import_practice_stats.py

```diff
@@ -25,12 +25,19 @@ def import_practice_stats(sqlite_path):
     if not os.path.exists(sqlite_path):
         raise RuntimeError('No SQLite file at: {}'.format(sqlite_path))
     connection = sqlite3.connect(sqlite_path)
+    dates = [date for (date,) in connection.execute('SELECT date FROM date')]
+    practice_statistics = get_practice_statistics_for_dates(dates)
+    write_practice_stats(connection, practice_statistics)
```

Comment:
Yes, I 100% agree. In general I try to avoid abbreviations even in function/variable names as I think it increases cognitive load when reading the code out of context. But I think sometimes `practice_statistics` just got a bit too unwieldy.